src/LanguageServer/ProtocolUnitTests/Options/SolutionAnalyzerConfigOptionsUpdaterTests.cs

```diff
@@ -23,7 +23,7 @@ public class SolutionAnalyzerConfigOptionsUpdaterTests
 {
     private static TestWorkspace CreateWorkspace()
     {
-        var workspace = new TestWorkspace(LspTestCompositions.LanguageServerProtocol
+        var workspace = new LspTestWorkspace(LspTestCompositions.LanguageServerProtocol
```

Comment:
With LspTestWorkspace - is it now possible to remove the windows only requirement from the LSP tests?  IIRC it was only there because the Editor test workspace targeted net472 or something like that?

definitely not asking for it in this PR, but wondering if I can do that in followups

Reply:
idk, feel free to try